src/components/LoginModal/index.tsx

```diff
@@ -31,22 +32,44 @@ export default function LoginModal({
   const navigate = useNavigate()
 
   const logIn = async ({ email, password }: ILoginFormState) => {
-    const { user } = await ApiSession.create(email, password)
+    const { user, error } = await ApiSession.create(email, password)
     if (user) {
       setUser(user)
       toast.success(t('success'))
       postLogin()
     } else {
-      toast.error(t('genericError', { ns: 'shared' }))
+      if (error == 'Unauthorized') {
+        toast(() => (
+          <LinkedTextInToast
+            regularText={t('loginFieldsError')}
+            linkedText={t('resetPasswordHelper')}
+            loginPage={LOGIN_PAGES.FORGOT_PASSWORD}
+            setPage={setPage}
```

Comment:
[Non-blocking suggestion] We could do away with the loginPage prop and just pass in the full set page function
```suggestion
            setPage={() => setPage(LOGIN_PAGES.FORGOT_PASSWORD}
```